spec/jquery/searching.spec.coffee

```diff
@@ -247,7 +247,7 @@ describe "Searching", ->
 
     expect(div.find(".active-result").length).toBe(1)
     expect(div.find(".active-result")[0].innerHTML).toBe("oh <em>h</em>ello")
-
+    
```

Comment:
Nit; unrelated change

Reply:
Yeah... not eactly a git wiz here, and the squahing turnied out to not be as easy as the instuctions suggested due to a conflict on the options page. I'm just about to give up and start a new branch instead.

Reply:
Ok the whole thing is a mess now. I have no idea how to recover. Can I start a new branch and pull request?